networking/multiple_networks/configuring-sr-iov.adoc

```diff
@@ -9,39 +9,54 @@ toc::[]
 == About SR-IOV hardware on {product-title}
 
 {product-title} includes the capability to use SR-IOV hardware on your nodes.
-You can attach SR-IOV virtual function (VF) interfaces to Pods on nodes with SR-IOV hardware.
+You can attach SR-IOV virtual function (VF) interfaces to Pods on nodes with
+SR-IOV hardware.
 
 You can use the {product-title} console to install SR-IOV by deploying the
 SR-IOV Network Operator. The SR-IOV Network Operator creates and manages the
-components of the SR-IOV stack. The Operator provides following features:
+components of the SR-IOV stack. The Operator provisions following components:
 
-* Discover the SR-IOV network device in cluster.
-* Initialize the supported SR-IOV NIC models on nodes.
-* Provision the SR-IOV network device plug-in on nodes.
-* Provision the SR-IOV CNI plug-in executable on nodes.
-* Provision the Network Resources Injector in cluster.
-* Manage configuration of SR-IOV network device plug-in.
-* Generate NetworkAttachmentDefinition custom resources (CR) for the SR-IOV CNI
-plug-in.
+* Provision the SR-IOV network operator deployment on master nodes.
+* Provision the SR-IOV network config daemon on worker nodes.
+* Provision the Operator webhook on master nodes.
+* Provision the Network resources injector on master nodes.
+* Provision the SR-IOV network device plug-in on worker nodes.
+* Provision the SR-IOV CNI plug-in executable on worker nodes.
 
 Here's the function of each above mentioned SR-IOV components.
 
+* The SR-IOV Operator is a Kubernetes Deployment that manages all SR-IOV
+components in cluster. It watches creation, update and deletion of Operator
+Custom Resources and takes corresponding actions such as generating
+NetworkAttachmentDefinition Custom Resources for SR-IOV CNI, creating and
+updating configuration of SR-IOV network device plug-in etc.
```

Comment:
Add a comma before "etc.":
"updating configuration of SR-IOV network device plug-in, etc."

Reply:
Done